lib/shopify_cli/commands/app/deploy.rb

```diff
@@ -2,7 +2,6 @@ module ShopifyCLI
   module Commands
     class App
       class Deploy < ShopifyCLI::Command::AppSubCommand
-        subcommand :Heroku, "heroku", "shopify_cli/commands/app/deploy/heroku"
```

Comment:
That command doesn't exist anymore. We didn't catch the bug because our tests were not going through this deploy command. Because the new logic for monorail traverses the entire command tree, this bug surfaced.

Reply:
Ah, now I see it's been moved to the Node/Rails/PHP services 👍